{{cookiecutter.repo_name}}/ci/templates/.travis.yml

```diff
@@ -1,9 +1,12 @@
-language: python
-python: '3.5'
```

Comment:
I'm not quite sure I understand you. What's the intention of doing that?

When using 3.5 Travis indeed tells "3.5 is not installed; attempting download", but then it runs as usual:
```console
$ python --version
Python 3.5.0
```